sormas-app/app/src/main/java/de/symeda/sormas/app/backend/report/AggregateReportDao.java

```diff
@@ -58,7 +58,13 @@ public List<AggregateReport> getReportsByEpiWeekAndUser(EpiWeek epiWeek, User us
 				public int compare(AggregateReport o1, AggregateReport o2) {
 					return o1.getDisease().toString().compareTo(o2.getDisease().toString());
 				}
-			});
+			}.thenComparing(new Comparator<AggregateReport>() {
+
+				@Override
+				public int compare(AggregateReport o1, AggregateReport o2) {
+					return o1.getAgeGroup().compareTo(o2.getAgeGroup());
```

Comment:
Will probably lead to a NullPointerException if one of the age groups is null (which is very likely to happen)